monkestation/code/modules/library/skill_learning/job_skillchips/shaft_miner.dm

```diff
@@ -0,0 +1,186 @@
+/*
+	DRG Style callout list.
+	Note the following:
+	Subtypes MUST go above their base types.
+	As in:
+			/mob/living/basic/mining/legion/snow/
+	ABOVE 	/mob/living/basic/mining/legion/
+
+	Typepaths must have a trailing forward slash.
+*/
+GLOBAL_LIST_INIT(miner_callouts, list(
```

Comment:
I think this can be a static list on the skillchip itself instead of global.

Reply:
Yeah, I'll shift it over. Was a global list for the sake of file formatting more than anything